### DISCUSSION TITLE
How to mask a portion of my graph and send the masked graph to my model?

### DISCUSSION THREAD

**Discussion by yangyangdotcom** [pyg-team/pytorch_geometric#6643] [Q&A] [answered]
Hi, I am new to this library and I would like to know what's the most effective way to mask my graph and send it to my model. I did a little bit of googling and find out `torch_geometric.utils.mask_select`, but not sure how to use it. Any help would be appreciated.

**Comment by rusty1s** [comment, marked as answer]
The most convenient way is `data.subgraph(node_mask)` :)

**Reply by yangyangdotcom** [reply]
Thanks!!!!
